packages/cli/src/commands/lint/LintCommand.ts

```diff
@@ -0,0 +1,63 @@
+import { Command } from 'commander'
+import kleur from 'kleur'
+import prettyjson from 'prettyjson'
+
+import { useGivenFileOrConfiguration } from '../../utils'
+import { lint } from './lint'
+
+/**
+ * Lint users scalar configs (scalar.config.json files)
+ */
+export function LintCommand() {
+  const cmd = new Command('lint')
+
+  cmd.description('Lint users scalar configs')
+  cmd.argument('[file]', 'File to lint')
+  cmd.action(async (inputArgument: string) => {
```

Comment:
I’d call the command `scalar check`, to not confuse it with validation/linting of OpenAPI documents.

Eventually the `scalar check` command could basically make sure everything is set up properly and in place, not just “lint” a file.

Reply:
I like this update! `check` sounds better and allows future flexibility